docs/installation/autoware/source-installation.md

```diff
@@ -20,30 +20,166 @@ sudo apt-get -y install git
+
+# If you don’t see an output line like the one above, then enable the Universe repository with these instructions.
+sudo apt install software-properties-common
+sudo add-apt-repository universe
+
+# Now add the ROS 2 apt repository to your system. First authorize our GPG key with apt.
+sudo apt update && sudo apt install curl gnupg lsb-release
+sudo curl -sSL https://raw.githubusercontent.com/ros/rosdistro/master/ros.key -o /usr/share/keyrings/ros-archive-keyring.gpg
+
+# Then add the repository to your sources list.
+echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/ros-archive-keyring.gpg] http://packages.ros.org/ros2/ubuntu $(source /etc/os-release && echo $UBUNTU_CODENAME) main" | sudo tee /etc/apt/sources.list.d/ros2.list > /dev/null
+
+# Update your apt repository caches after setting up the repositories.
+sudo apt update
+
+# Desktop Install
+sudo apt install ros-galactic-desktop
+
+# Environment setup
+# (Optional) You can source ros2 galactic in the ~/.bashrc file.
+echo 'source /opt/ros/galactic/setup.bash' >> ~/.bashrc
+```
+
+Install ROS2 Dev Tools:
+
+```bash
+# Taken from https://docs.ros.org/en/galactic/Installation/Ubuntu-Development-Setup.html
+sudo apt update && sudo apt install -y \
+  build-essential \
+  cmake \
+  git \
+  python3-colcon-common-extensions \
+  python3-flake8 \
+  python3-pip \
+  python3-pytest-cov \
+  python3-rosdep \
+  python3-setuptools \
+  python3-vcstool \
+  wget
+
+# Install some pip packages needed for testing
+python3 -m pip install -U \
+  flake8-blind-except \
+  flake8-builtins \
+  flake8-class-newline \
+  flake8-comprehensions \
+  flake8-deprecated \
+  flake8-docstrings \
+  flake8-import-order \
+  flake8-quotes \
+  pytest-repeat \
+  pytest-rerunfailures \
+  pytest \
+  setuptools
+
+# Initialize rosdep
+sudo rosdep init
+rosdep update
+```
+
+Install the `rmw_cyclonedds_cpp` RMW Implementation:
+
+```bash
+# For details: https://docs.ros.org/en/galactic/How-To-Guides/Working-with-multiple-RMW-implementations.html
+sudo apt update
+sudo apt install ros-galactic-rmw-cyclonedds-cpp
+
+# (Optional) You set the default RMW implementation in the ~/.bashrc file.
+echo 'export RMW_IMPLEMENTATION=rmw_cyclonedds_cpp' >> ~/.bashrc
+```
+
+Install pacmod:
+
+```bash
+# Taken from https://github.com/astuff/pacmod3#installation
+sudo apt install apt-transport-https
+sudo sh -c 'echo "deb [trusted=yes] https://s3.amazonaws.com/autonomoustuff-repo/ $(lsb_release -sc) main" > /etc/apt/sources.list.d/autonomoustuff-public.list'
+sudo apt update
+sudo apt install ros-galactic-pacmod3
+```
+
+Install rest of the dependencies:
+
+```bash
+# Install gdown to download files from CMakeLists.txt
+pip3 install gdown
+
+sudo apt install geographiclib-tools
+
+# Add EGM2008 geoid grid to geographiclib
+sudo geographiclib-get-geoids egm2008-1
+
+pip3 install pre-commit clang-format
+
+# Install Golang (Add Go PPA for shfmt)
+sudo add-apt-repository ppa:longsleep/golang-backports
```

Comment:
@xmfcx @kenji-miyake I've submitted https://github.com/tier4/autoware-spell-check-dict/pull/258 to fix the spelling warnings